man/runc-features.8.md

```diff
@@ -0,0 +1,14 @@
+% runc-features "8"
+
+# NAME
+**runc-features** - lists containers
+
+# SYNOPSIS
+**runc features**
+
+# DESCRIPTION
+The **features** command show enabled features. If you don't know what
+is feature you can check feature's section in runtime-spec in github.
```

Comment:
This sounds condescending to a reader. It's better to give a brief overview and refer to more documentation.

Reply:
sure. i'm gonna explain features a little bit and provide more documentation

Reply:
Is it ok now?